frontend/pages/policies/PolicyPage/components/PolicyQueriesErrorsListWrapper/PolicyQueriesErrorsListWrapper.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { noop } from "lodash";
+
+import { IHostPolicyQueryError } from "interfaces/host";
+import TableContainer from "components/TableContainer";
+import {
+  generateTableHeaders,
+  generateDataSet,
+} from "./PolicyQueriesErrorsTableConfig";
+
+const baseClass = "policies-queries-list-wrapper";
+const noPolicyQueries = "no-policy-queries";
```

Comment:
add space for readability?

Reply:
I'm not sure what you mean. Can you provide an example?

Reply:
👾